src/qt/optionsmodel.cpp

```diff
@@ -151,10 +151,32 @@ void OptionsModel::Init(bool resetSettings)
     language = settings.value("language").toString();
 }
 
+/** Helper function to copy contents from one QSettings to another.
+ * By using allKeys this also covers nested settings in a hierarchy.
+ */
+static void copySettings(QSettings &dst, const QSettings &src)
```

Comment:
Don't see the benefit of this function, move the code to `backupSettings()`? Otherwise `&` next to type name.

Reply:
The point is that this is a general function, it could be also e.g. used to restore QSettings from a backup, if that is ever necessary. Would prefer to keep it. Better too much factoring than too little.

Reply:
Ok then, just update to `CopySettings` and move `&`.